src/PreCharge/PreCharge.cpp

```diff
@@ -257,10 +276,13 @@ uint16_t PreCharge::getObjectDictionarySize() {
 }
 
 void PreCharge::sendChangePDO() {
-    uint8_t value = (IOStatus << 8) | Statusword;
-    uint8_t payload[1] = {value};
-    IO::CANMessage changePDOMessage(0, 6, payload, false);
+    uint8_t payload[] = {Statusword, 0x00, static_cast<unsigned short>(keyInStatus) << 4 | static_cast<unsigned short>(stoStatus),
+                                           static_cast<unsigned short>(batteryOneOkStatus) << 4  | static_cast<unsigned short>(batteryTwoOkStatus),
+                                           static_cast<unsigned short>(eStopActiveStatus) << 4 | static_cast<unsigned short>(forwardStatus),
+                                           static_cast<unsigned short>(pcStatus) << 4 | static_cast<unsigned short>(dcStatus),
+                                           static_cast<unsigned short>(contStatus) << 4 | static_cast<unsigned short>(apmStatus)};
```

Comment:
```suggestion
                                           static_cast<unsigned short>(contStatus) << 4 | static_cast<unsigned short>(apmStatus),};
```
Adding a comma here will fix the auto-formatting